hpx/runtime/threads/detail/scheduling_loop.hpp

```diff
@@ -559,23 +547,18 @@ namespace hpx { namespace threads { namespace detail
         bool may_exit = false;
 
 #if defined(HPX_HAVE_NETWORKING)
-        bool networking_is_enabled = hpx::is_networking_enabled();
```

Comment:
What is the rationale of enabling background threads if networking is disabled?

Reply:
It's not enabled, the logic has just moved up a bit (to avoid the call to `hpx::is_networking_enabled` and thus a dependency on the runtime configuration). I instead set the number of background threads to 0 in the thread manager if networking is disabled with the same effect.